src/components/dialog/dialog.scss

```diff
@@ -80,7 +80,7 @@ md-dialog {
     }
   }
 
-  .md-actions, md-dialog-actions {
```

Comment:
we should leave the style classes for deprecated support (while we removing the warnings).
Otherwise this is a breaking change.